src/table/utils/styling-utils.js

```diff
@@ -153,6 +153,28 @@ export function getBodyCellStyle(layout, theme) {
   };
 }
 
+/**
+ * Get total head cell style
+ *
+ * @param {Object} totalHeaderStyle
+ * @param {Object} layout
+ * @returns total cell style regarding the total head position
+ */
+export function getTotalsCellStyle(totalHeaderStyle, layout) {
+  if (getTotalPosition(layout) === 'noTotals') return false;
```

Comment:
better to pass the position from totals component, you should have access to it there

Reply:
then you don't need layout